bundles/org.openhab.core.io.websocket/src/main/java/org/openhab/core/io/websocket/audiopcm/PCMWebSocketStreamIdUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.websocket.audiopcm;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.SecureRandom;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Utils to read/write the audio packets send though the websocket binary protocol.
+ *
+ * @author Miguel Álvarez Díez - Initial contribution
+ */
+@NonNullByDefault
+public class PCMWebSocketStreamIdUtil {
```

Comment:
As this is a util class, I personally would make it final and make the default ctor private.